ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -871,11 +871,11 @@ ethernet_interfaces:
     ipv6_nd_ra_disabled: < true | false >
     ipv6_nd_managed_config_flag: < true | false >
     ipv6_nd_prefixes:
-      < IPv6_address_1/Mask >:
+      - ipv6_mask: < IPv6_address_1/Mask >
```

Comment:
Please set this name to ```ipv6_address``` or ```ipv6_address_prefix```. Any comments @ClausHolbechArista

Reply:
I prefer ipv6_address

Reply:
fixed

Reply:
Sorry to be late - since the list is called ipv6_nd_prefixes, I would prefer `prefix` as the key.

Reply:
Went through this with @ankudinov too, and we have settled on `ipv6_prefix`. @Shivani-chourasiya can I ask you to update again. Also remember the readme :)

Reply:
Updated!